pandas/io/json/_table_schema.py

```diff
@@ -211,7 +211,7 @@ def build_table_schema(data, index=True, primary_key=None, version=True):
 
     Notes
     -----
-    See `_as_json_table_type` for conversion types.
+    See [`Table Schema`](https://pandas.pydata.org/docs/user_guide/io.html#table-schema) for conversion types. # noqa: E501
```

Comment:
I don't think this will render correctly in rst: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Also to pass flake8 you can just wrap this in the next line like this example:

```
    To learn more about the frequency strings, please see `this link
    <https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases>`__.
```

Reply:
My bad, I thought Markdown was being used

Reply:
If you wish to use an alias - see the spinx docs. https://sphinxtechnicalwriting.readthedocs.io/en/latest/sphinx/linking.html

`gbq.py` for an example.

You can build the documentation locally - https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#how-to-build-the-pandas-documentation to check that it renders as expected. (Feel free to post a screenshot on PRs)

Thanks